examples/multi_modality/predict_modality/scmogcn.py

```diff
@@ -33,6 +33,8 @@ def pipeline(inductive=False, verbose=2, logger=None, **kwargs):
     mod2 = anndata.concat((modalities[1], modalities[3]))
     mod1.var_names_make_unique()
     mod2.var_names_make_unique()
+    mod1.obs["batch"] = mod1.obs["batch"].cat.add_categories("unknown").fillna("unknown")
+    mod2.obs["batch"] = mod2.obs["batch"].cat.add_categories("unknown").fillna("unknown")
```

Comment:
@xingzhongyu this will raise an error if `"unknown"` is already in the batch obs, please work around this to improve compatibility with user defined data

Reply:
Thanks,I have changed it